tox.ini

```diff
@@ -1,39 +1,47 @@
 [tox]
-envlist = py27,py36,static,pidiff
+envlist = py27,py36
+skipsdist = True
 
 [testenv]
-deps=-rtest-requirements.txt
-commands=pytest -v {posargs}
-whitelist_externals=sh
-
-[testenv:py27]
-deps=
-  # Note: we need to explicitly list requirements.txt here
-  # so it's processed at the same time as the constraints file
-  -rrequirements.txt
-  -rtest-requirements.txt
+deps =
+    -rrequirements.txt
+    -rtest-requirements.txt
+commands =
+    python setup.py install
```

Comment:
I don't think you need to install the package to test it.

Reply:
Huh. You're right. I assumed this was necessary with sdist deactivated (due to same python3 default issue).